src/main/webapp/WEB-INF/tags/bodyFooter.tag

```diff
@@ -1,13 +1,14 @@
 <%@ tag description="Body footer (bottom of page)" %>
 <%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
 <%@ tag import="teammates.common.util.Config" %>
+<%@ attribute name="isAdmin" %>
 <div id="footerComponent" class="container-fluid">
     <div class="container">
         <div class="row">
             <div class="col-md-2">
                 <span>[<a href="/index.html">TEAMMATES</a> V<%= Config.inst().getAppVersion() %>]</span>
             </div>
-            <div class="col-md-8">
+            <div class="col-md-8" <c:if test="${isAdmin}">id="admin-institute"</c:if>>
```

Comment:
just one minor suggestion here. The standard is to use camel casing for ids and small letters with dashes for classes. See if we can change this to follow them